### DISCUSSION TITLE
Export

### DISCUSSION THREAD

**Discussion by Priyanka-sharma181** [postmanlabs/postman-flows#319] [General]
Export option is not there we can not export

**Comment by samay-v** [comment]
Hi @Priyanka-sharma181, 
Could you share your use-case for exporting. 

If you wish to share your flows with others, you could try [sharing](https://learning.postman.com/docs/collaborating-in-postman/sharing/) or [forking](https://learning.postman.com/docs/collaborating-in-postman/version-control/#forking-postman-entities)
